src/App.tsx

```diff
@@ -15,17 +15,20 @@ import NextButton from './components/shared/nextButton';
 import Sidebar from './components/shared/sidebar';
 
 function App(): JSX.Element {
+  const [enabled, setEnabled] = useState(false);
+  const enable = () => {setEnabled(true);};
+  const disable = () => {setEnabled(false);};
   return (
     <Router>
       <Sidebar/>
       <div id = "contents">
         <Switch>
           <Route exact path="/"> <Landing/> </Route>
           <Route path="/variable"> <Variable/> </Route>
-          <Route path="/python"> <Python/> </Route>
+          <Route path="/python"> <Python enable={enable}/> </Route>
```

Comment:
nit (feel free to dismiss): i think you can just embed `enable` and `disable` since it's only a line. 

Generally, I like to throw it in directly if it's only a line, and take it out if it's longer (or has the potential to need to be longer).